<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="hdf5FolderPath" default="/seerep/seerep-data/cognitiveWeeding/uos"/>
    <arg name="jsonFilePath" default="/seerep/seerep-data/cognitiveWeeding/uos/uos_test.json"/>

    <node name="seerep_ros_communication_jsonPointDumper" pkg="seerep_ros_communication" type="seerep_ros_communication_jsonPointDumper" output="screen">
        <rosparam param="hdf5FolderPath" subst_value="True">$(arg hdf5FolderPath)</rosparam>
        <rosparam param="jsonFilePath" subst_value="True">$(arg jsonFilePath)</rosparam>
    </node>
</launch>
